vision/imageloader/imageloader.go

```diff
@@ -73,10 +82,22 @@ func New(fn string, vocab map[string]int, trans *transforms.ComposeTransformer,
 		bufSize:     bufSize,
 		seed:        seed,
 	}
-	runtime.LockOSThread()
-	go m.readSamples()
 	go m.shuffleSamples()
-	go m.samplesToMinibatches()
+	// `readSamples` and `samplesToMinibatches` calls `gocv`, we make them only run in background OS threads.
```

Comment:
Make we should add some detailed comment, or add a issue linke.

Reply:
Done.